### ISSUE TITLE
energy consumption graphs on Mac, prefer bar graph vs smooth curves

### ISSUE THREAD

**Issue by wjcroft** [tnmendes/watt#48]
Tiago, hi. 

Thanks for your great apps. I don't have an iOS device, so I use your Mac app. I notice on your website demo images, bar graphs are used for daily power consumption totals. Yet on this Mac version, instead of a bar graph, a kind of overly-smoothed curve is shown, which appears (to me) to lose all the detail and fine changes. It just looks smoothed / smashed out excessively, so very hard to see day to day changes as significantly different values. I'm pretty certain I am missing significant day to day changes, by the heavy amount of curve smoothing being applied.

https://www.google.com/search?q=curve+smoothing+algorithm

Request: could you please revert the Mac version to use the same bar graphs as shown on your website images? The vertical scale is useful, but hard to interpolate with the smoothing.

http://watt-app.com/

Regards,

William Croft

**Comment by tnmendes** [comment]
Hey William,

The website is still using old images from the old design. 

More or less 6 months ago I did a major redesign of the application and one of the things I changed was the way I presented the graphics to make it more visually appealing to new users and also to easily show consumption and runtime on the same graph.

I admit that doing this made reading the data I bit more complicated. I  have to look at the code and see how complicated it is to have the option to select the type of graphics but right now I am working on other new features so this will have to go to my backlog list.

**Comment by wjcroft** [comment]
Tiago, thanks.

Am I correct that the consumption info is only recorded DAILY? The curved graphs make it appear that measurements are continuous. This is not the case. A very easy fix would be to just draw straight line segments between days on the graphs. Then the bar graph would really not be needed.

Thanks,

**Comment by wjcroft** [comment]
I just realized that I can press and hold the mouse button down, while moving back and forth on the consumption graph. So this shows clearly what the actual day-by-day logged daily usage is, and how that corresponds to the smoothed graph points.

Maybe my request for straight line segments is now superfluous, now that I can see the points clearly with this trick. I still think it would add clarity to the graphs. Maybe this is all obvious to other users, but it was not to me. I did go looking for help, but it's not in the menu.

Regards, William
